### PR TITLE
Cleanup playground FAQ and add inline examples.

### PR DESCRIPTION
**Pull request by evanlally**
Updates in progress

image

### CODE REVIEW

File: playgroundFaq.md [hunk 1]
```diff
@@ -9,61 +9,121 @@ Have you already integrated CHAPI into your project and now you’re interested
...
+If you can provide a [VC-API /issue endpoint](https://w3c-ccg.github.io/vc-api/#issue-credential), we can add your issuer to the list of options accessible from the "Gear" icon on the site.  Your endpoint should end in `/credentials/issue`.
 
-We've put together a [quick test](https://github.com/credential-handler/chapi-issuer-test-suite) that you can run locally against your endpoint (the test runs locally; the endpoint can be running anywhere).  If your issuer endpoint can pass the first three test conditions, then we'll be ready to add it to the playground.
+We've put together a [simple test utility](https://github.com/credential-handler/CHAPI Playground-test-suite) that you can run locally against your endpoint (the test runs locally; the endpoint can be running anywhere).  If your issuer endpoint can pass the first three test conditions, then we'll be ready to add it to the list of options in the CHAPI Playground.
```
Comment (davidlehn) [inline playgroundFaq.md:16]: Fix link.
```suggestion
We've put together a [simple test utility](https://github.com/credential-handler/chapi-playground-test-suite) that you can run locally against your endpoint (the test runs locally; the endpoint can be running anywhere).  If your issuer endpoint can pass the first three test conditions, then we'll be ready to add it to the list of options in the CHAPI Playground.
```

File: playgroundFaq.md [hunk 2]
```diff
@@ -9,61 +9,121 @@ Have you already integrated CHAPI into your project and now you’re interested
...
 
-Example data for each step can be found [here](https://gist.github.com/tolson4/6e44720a15957136fbe6ca33403c4bcb).
+In this example, the `interact` field describes the type and URL of the /exchanges endpoint.  The `type` is an `UnmediatedPresentationService2021`, which is outlined in the [VCAPI spec](https://w3c-ccg.github.io/vc-api/#exchange-examples).
 
-We’ve added an example to the [Verifiable Presentation Request (VPR) spec](https://w3c-ccg.github.io/vp-request-spec/#example-example-interact-request) to show how an issuer can ask for DID Authentication and then point the wallet at /exchanges endpoint to perform the Auth-Pickup workflow.
+The CHAPI Playground implementation follows the general pattern, via these spcific steps (with example data):
+
+**Step 1**: CHAPI Playground sends a DID Auth Request (VPR) over CHAPI to the wallet.  This includes an `interact` field that specifies an /exchanges endpoint on the CHAPI Playground.  
+```
```
Comment (davidlehn) [inline playgroundFaq.md:62]:
````suggestion
```json
````

File: playgroundFaq.md [hunk 3]
```diff
@@ -9,61 +9,121 @@ Have you already integrated CHAPI into your project and now you’re interested
...
+{
+    "query": {
```
Comment (davidlehn) [inline playgroundFaq.md:64]: Use 2 space indents in this block to match other examples.

File: playgroundFaq.md [hunk 4]
```diff
@@ -9,61 +9,121 @@ Have you already integrated CHAPI into your project and now you’re interested
...
+            }
+        ]
+    },
+    "challenge": "g2Wvqg7-cQGWYCCH55rUl",
+    "domain": "https://playground.chapi.io"
+}
+```
+
+**Step 2**: The Wallet POSTs a VP containing the DID Auth to the /exchanges endpoint.
+```
```
Comment (davidlehn) [inline playgroundFaq.md:84]:
````suggestion
```json
````

File: playgroundFaq.md [hunk 5]
```diff
@@ -9,61 +9,121 @@ Have you already integrated CHAPI into your project and now you’re interested
...
+  "verifiablePresentation" :{
+    "@context": [
+      "https://www.w3.org/2018/credentials/v1"
+    ],
+    "type": [
+      "VerifiablePresentation"
+    ],
+    "holder": "did:example:z6MkjsQSCqdN4CGE6R9tKhETAEoPYdcpO5v4tK2USAhWptpr",
+    "proof": {
+      "type": "Ed25519Signature2020
```
Comment (davidlehn) [inline playgroundFaq.md:95]:
```suggestion
      "type": "Ed25519Signature2020",
```

File: playgroundFaq.md [hunk 6]
```diff
@@ -9,61 +9,121 @@ Have you already integrated CHAPI into your project and now you’re interested
...
+      "challenge": "TTkKgear2sBt4TYxr_zrL",
+      "domain": "https://playground.chapi.io",
+      "proofValue": "eyJhbGciOiJFZERTQSIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..gaG1RaxRy-xLkjVN-7ulh0dbUU1EX"
+    }
+  }
+}
+```
+
+**Step 3**: The CHAPI Playground /exchanges endpoint responds with a VP containing the specified credential
+```
```
Comment (davidlehn) [inline playgroundFaq.md:108]: Using `jsonc` due to comment
````suggestion
```jsonc
````

File: playgroundFaq.md (hunk 1 above)
Comment (evanlally) [reply playgroundFaq.md:16]: Thanks @davidlehn !

File: playgroundFaq.md (hunk 6 above)
Comment (evanlally) [reply playgroundFaq.md:108]: Formatting looks weird with our jekyll template - keeping as a generic code block.

File: playgroundFaq.md (hunk 4 above)
Comment (evanlally) [reply playgroundFaq.md:84]: Formatting looks weird with our jekyll template - keeping as a generic code block.

File: playgroundFaq.md [hunk 7]
```diff
@@ -9,61 +9,121 @@ Have you already integrated CHAPI into your project and now you’re interested
 
 ## Issuers
 ### New issuer introduction:
```
Comment (gannan08) [inline playgroundFaq.md:11]:
```suggestion
### Introduction
```

File: playgroundFaq.md [hunk 8]
```diff
@@ -9,61 +9,121 @@ Have you already integrated CHAPI into your project and now you’re interested
...
-Authentication is up-to-you.  We are working with some open (unauthenticated) VC-API endpoints now, but we also have support for zCap and Oauth2 authentication.  You can test with the endpoint open and then add backend auth as a second step, if you would like to have it.
+Issuer endpoint authentication is up-to-you.  Many issuers choose to start with open (unauthenticated) VC-API endpoints, but the CHAPI Playground backend also supports zCap and Oauth2 authentication.  You can test with the endpoint open and then add backend auth as a second step, if you would like to have it.
 
 ### Do I need to enable CORS on my issuer endpoint?
-You do not need CORS because we are just making a backend call from the CHAPI-playground web app to your VC-API endpoint.
+Issuer backends do not need CORS because the CHAPI Playground web app makes a backend call to your VC-API endpoint.
 
 
 ## Wallets
 ### New wallet introduction
```
Comment (gannan08) [inline playgroundFaq.md:34]:
```suggestion
### Introduction
```

File: playgroundFaq.md [hunk 9]
```diff
@@ -9,61 +9,121 @@ Have you already integrated CHAPI into your project and now you’re interested
...
+        "type": "VerifiableCredentialApiExchangeService",
+        "serviceEndpoint": "https://playground.chapi.io/exchanges/eyJjcmVkZW50aWFsIjoiaHR0cHM6Ly9wbGF5Z3JvdW5kLmNoYXBpLmlvL2V4YW1wbGVzL2pmZjIvamZmMi5qc29uIiwiaXNzdWVyIjoiZGIvdmMifQ/esOGVHG8d44Q"
+      },
+      {
+        "type": "CredentialHandlerService"
+      }
+    ]
+  },
+  "challenge": "g2Wvqg7-cQGWYCCH55rUl",
+  "domain": "https://playground.chapi.io"
```
Comment (gannan08) [inline playgroundFaq.md:79]:
```suggestion
  "domain": "playground.chapi.io"
```

Protocols are not included in domains.

File: playgroundFaq.md [hunk 10]
```diff
@@ -9,61 +9,121 @@ Have you already integrated CHAPI into your project and now you’re interested
...
+      "VerifiablePresentation"
+    ],
+    "holder": "did:example:z6MkjsQSCqdN4CGE6R9tKhETAEoPYdcpO5v4tK2USAhWptpr",
+    "proof": {
+      "type": "Ed25519Signature2020",
+      "created": "2022-10-17T18:12:55Z",
+      "verificationMethod": "did:example:z6MkjsQSCqdN4CGE6R9tKhETAEoPYdcpO5v4tK2USAhWptpr#z6MkjsQSCqdN4CGE6R9tKhETAEoPYdcpO5v4tK2USAhWptpr",
+      "proofPurpose": "authentication",
+      "challenge": "TTkKgear2sBt4TYxr_zrL",
+      "domain": "https://playground.chapi.io",
```
Comment (gannan08) [inline playgroundFaq.md:100]:
```suggestion
      "domain": "playground.chapi.io",
```

Protocols are not included in domains.

**Review by gannan08 — CHANGES_REQUESTED** [review]

File: playgroundFaq.md [hunk 11]
```diff
@@ -8,62 +8,122 @@ permalink: /developers/playgroundfaq
 Have you already integrated CHAPI into your project and now you’re interested in making your Issuer, Wallet, or Verifier available for experimentation? This Integration Guide/FAQ for the CHAPI Playground will help you accomplish this!
 
 ## Issuers
-### New issuer introduction:
-You can integrate your Issuer into our tooling with the [CHAPI-Playground Issuer](https://playground.chapi.io/issuer).
+### Introduction
+You can integrate your Issuer into our tooling with the [CHAPI Playground Issuer](https://playground.chapi.io/issuer).
 
-If you can provide a [VC-API /issue endpoint](https://w3c-ccg.github.io/vc-api/#issue-credential), we can add your issuer to the list of options accessible from the "Gear" icon on the site.  That should be all we need for integration with the Veres Wallet.  (Note that you'll also be able to use this site to demonstrate issuance with any other CHAPI-enabled wallets).
+If you provide a [VC-API /issue endpoint](https://w3c-ccg.github.io/vc-api/#issue-credential), we can add your issuer to the list of options accessible from the "Gear" icon on the site.  Your endpoint should end in `/credentials/issue`.
```
Comment (gannan08) [inline playgroundFaq.md:14]:
```suggestion
If you provide a [VC-API Issue Credential Endpoint](https://w3c-ccg.github.io/vc-api/#issue-credential), we can add your issuer to the list of options accessible from the "Gear" icon on the site.  Your endpoint should end in `/credentials/issue`.
```

File: playgroundFaq.md [hunk 12]
```diff
@@ -8,62 +8,122 @@ permalink: /developers/playgroundfaq
...
+
+First, the Playground will send a Verifiable Presentation Request as a "web" credential object using `navigator.credentials.get()`.  Second, your wallet will respond with a Verifiable Presentation,  also as a "web" credential object.  Examples of both the VPR and VP are included in the [wallet developer notes](https://chapi.io/developers/wallets).
+
+The Playground receives the DID Auth, it extracts the value of `holder` and inserts it into the credential being issued.  Then, the credential is issued using the basic CHAPI `store` workflow.
+
+
+### Integrating with the VC-API/exchanges workflow
+The general /exchanges pattern is illustrated in the [Verifiable Presentation Request (VPR) spec](https://w3c-ccg.github.io/vp-request-spec/#example-example-interact-request), which shows how an issuer can ask for DID Authentication and then point the wallet at /exchanges endpoint to perform the Auth-Pickup workflow.
+
+In this example, the `interact` field describes the type and URL of the /exchanges endpoint.  The `type` is an `UnmediatedPresentationService2021`, which is outlined in the [VC-API spec](https://w3c-ccg.github.io/vc-api/#exchange-examples).
```
Comment (gannan08) [inline playgroundFaq.md:57]:
```suggestion
In this example, the `interact` field describes the type and URL of the `/exchanges` endpoint.  The `type` is an `UnmediatedPresentationService2021`, which is outlined in the [VC-API spec](https://w3c-ccg.github.io/vc-api/#exchange-examples).
```

**Review by gannan08 — CHANGES_REQUESTED** [review]

File: playgroundFaq.md [hunk 13]
```diff
@@ -8,62 +8,122 @@ permalink: /developers/playgroundfaq
...
+
+
+### Integrating with the VC-API Exchanges workflow
+The general `/exchanges` pattern is illustrated in the [Verifiable Presentation Request (VPR) spec](https://w3c-ccg.github.io/vp-request-spec/#example-example-interact-request), which shows how an issuer can ask for DID Authentication and then point the wallet at `/exchanges` endpoint to perform the Auth-Pickup workflow.
+
+In this example, the `interact` field describes the type and URL of the `/exchanges` endpoint.  The `type` is an `UnmediatedPresentationService2021`, which is outlined in the [VC-API spec](https://w3c-ccg.github.io/vc-api/#exchange-examples).
+
+The CHAPI Playground implementation follows the general pattern, via the following spcific steps (with example data):
+
+**Step 1**: CHAPI Playground sends a DID Auth Request (VPR) over CHAPI to the wallet.  This includes an `interact` field that specifies an /exchanges endpoint on the CHAPI Playground.
```
Comment (gannan08) [inline playgroundFaq.md:61]:
```suggestion
**Step 1**: CHAPI Playground sends a DID Auth Request (VPR) over CHAPI to the wallet.  This includes an `interact` field that specifies an `/exchanges` endpoint on the CHAPI Playground.  
```

File: playgroundFaq.md [hunk 14]
```diff
@@ -8,62 +8,122 @@ permalink: /developers/playgroundfaq
...
+      "proofPurpose": "authentication",
+      "challenge": "TTkKgear2sBt4TYxr_zrL",
+      "domain": "playground.chapi.io",
+      "proofValue": "eyJhbGciOiJFZERTQSIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..gaG1RaxRy-xLkjVN-7ulh0dbUU1EX"
+    }
+  }
+}
+```
+
+**Step 3**: The CHAPI Playground /exchanges endpoint responds with a VP containing the specified credential
```
Comment (gannan08) [inline playgroundFaq.md:107]:
```suggestion
**Step 3**: The CHAPI Playground `/exchanges` endpoint responds with a VP containing the specified credential
```

**Review by gannan08 — APPROVED** [review]

**Pull request merged**